### ISSUE TITLE
snapshotCreation task included in snapshot

### ISSUE THREAD

**Issue by savikko** [meilisearch/MeiliSearch#4653]
Hi!

We have about 30G index with something like 2M documents here. When we are creating snapshot there (it takes about 30 minutes or so, just as an info), and when we are importing it to another instance, new instance was taking 100% CPU a while after start.

Then we found out, that even if we start this new instance without any snapshot parameter, there is still snapshotCreation -task there, with status `processing`.

Of course, we can cancel the tasks (seems to take a while) but it would be best, that that task would not be there in snapshot waiting to be launched on new instance.

Or, are we probably doing something wrong here?

We are launching snapshot via API and then importing it via `--import-snapshot`.

**Comment by Kerollmops** [comment]
Hey @savikko 👋,

Thank you very much for the report. It's an interesting bug report that we didn't notice before, even if it's been the behavior for a long time. You are using the snapshots and generating them the right way.

We need to figure out the best way to fix this issue. I would prefer we do the snapshot like we do, with the enqueued `snapshotCreation` task, but modify the snapshot afterward. Marking it as processed so we don't process it again when loading the snapshot.

I triaged this issue, and we will prioritize the work. Thank you again and have a good day 🌞

**Comment by savikko** [comment]
Yep, I think that would be way to go as this is kind of "special" case.

Also, as a workaround we are now setting snapshot output dir as something which is not writable so snapshotCreation just fails and we don't have to cancel that.

Thanks for everything, and have a good day also there! :)
